README.md

```diff
@@ -0,0 +1,143 @@
+
+
+# Cisco IOS Collection
+[![CI](https://zuul-ci.org/gated.svg)](https://dashboard.zuul.ansible.com/t/ansible/project/github.com/ansible-collections/cisco.ios) <!--[![Codecov](https://img.shields.io/codecov/c/github/ansible-collections/vyos)](https://codecov.io/gh/ansible-collections/cisco.ios)-->
+
+The Ansible Cisco IOS collection includes a variety of Ansible content to help automate the management of Cisco IOS network appliances.
+
+This collection has been tested against Cisco IOSv version 15.2 on VIRL.
+
+### Supported connections
+The Cisco IOS collection supports ``network_cli``  connections.
+
+## Included content
+
+Click the ``Content`` button to see the list of content included in this collection.
+
+## Installing this collection
+
+You can install the Cisco IOS collection with the Ansible Galaxy CLI:
+
+    ansible-galaxy collection install cisco.ios
+
+You can also include it in a `requirements.yml` file and install it with `ansible-galaxy collection install -r requirements.yml`, using the format:
+
+```yaml
+---
+collections:
+  - name: cisco.ios
+    version: 0.0.3
+```
+## Using this collection
+
+
+This collection includes [network resource modules](https://docs.ansible.com/ansible/latest/network/user_guide/network_resource_modules.html).
+
+### Using modules from the Cisco IOS collection in your playbooks
+
+You can call modules by their Fully Qualified Collection Namespace (FQCN), such as `cisco.ios.ios_l2_interfaces`.
+The following example task replaces configuration changes in the existing configuration on a Cisco IOS network device, using the FQCN:
+
+```yaml
+---
+  - name: Replace device configuration of specified L2 interfaces with provided configuration.
+  cisco.ios.ios_l2_interfaces:
+    config:
+      - name: GigabitEthernet0/2
+        trunk:
+        - allowed_vlans: 20-25,40
+          native_vlan: 20
+          pruning_vlans: 10
+          encapsulation: isl
+    state: replaced
+
+```
+
+Alternately, you can call modules by their short name if you list the `cisco.ios` collection in the playbook's `collections`, as follows:
+
+```yaml
+---
+- hosts: ios01
+  gather_facts: false
+  connection: network_cli
+
+  collections:
+    - cisco.ios
+
+  tasks:
+    - name: Delete provided configuration from the device configuration
+      ios_static_routes:
```

Comment:
we can take any of ios_ospv2 delete operation as example or overridden/replaced state example. As I'll be updating the static_routes delete operation guess by this week

Reply:
I don't see ios_ospfv2 merged yet, so I took another L3 interfaces example.

Reply:
cool +1